controllers/chat/group/removeGroupMember.js

```diff
@@ -41,6 +41,9 @@ const removeGroupMember = async (req, res) => {
         other_text: textTargetUser,
         type: 'system',
         user_id: userId,
+        only_to_user_show: userId,
+        disable_to_user: false,
+        is_from_prepopulated: true,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of new properties `only_to_user_show`, `disable_to_user`, and `is_from_prepopulated` to the message object seems fine. However, make sure these properties are used correctly elsewhere in the codebase and their values are set appropriately. For instance, `only_to_user_show` is set to `userId` which seems like it should be a boolean value based on its name. Please verify this.
```diff
+ only_to_user_show: userId,
+ disable_to_user: false,
+ is_from_prepopulated: true,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->